app/views/layouts/application.html.erb

```diff
@@ -18,6 +18,9 @@
 </head>
 <body>
   <%= render 'layouts/header_html' %>
+  <% flash.each do |message_type, message| %>
```

Comment:
(note) If you're using devise, it can add an extra entry in the flash hash which we don't want to display, so it's usually better to enumerate the levels exlicitly:
```erb
<% %i[notice alert].each do |type| %>
  <%= tag.div flash[type], class: "... alert-#{type}" if flash[type].present? %>
<% end %>